plugins/plugins.go

```diff
@@ -691,6 +691,11 @@ func (m *Manager) Reconfigure(config *config.Config) error {
 		}
 	}
 
+	// don't erase persistence directory
+	if config.PersistenceDirectory == nil {
+		config.PersistenceDirectory = m.Config.PersistenceDirectory
+	}
+
```

Comment:
@blacksails if we want to preserve the persistence directory I think the only change we need should be

```go
config.PersistenceDirectory = m.Config.PersistenceDirectory // don't overwrite persistence directory
```